sherpa/astro/instrument.py

```diff
@@ -228,9 +231,11 @@ def filter(self):
             # If PHA grid is in angstroms then convert to keV for
             # consistency
             if (bin_lo[0] > bin_lo[-1]) and (bin_hi[0] > bin_hi[-1]):
-                bin_lo, bin_hi = DataPHA._hc/pha.bin_hi, DataPHA._hc/pha.bin_lo
+                bin_lo, bin_hi = DataPHA._hc / \
```

Comment:
Rather than have the awkward  continuation line, why not just break it into two lines (which I think also emphasises better what is going on):

    bin_lo = DataPHA._hc / pha.bin_hi
    bin_hi = DataPHA._hc / pha.bin_lo